packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
+
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+export let title = '';
+export let message = '';
+export let variant: Variants = 'info';
+export let progress = 1;
+export let exitable = false;
+
+$: isExitable = exitable;
+
+const dispatch = createEventDispatcher();
```

Comment:
Can you please add typings and docstrings? [Example](https://github.com/viamrobotics/prime/blob/main/packages/core/src/lib/pill.svelte#L26-L29)